config/config.toml

```diff
@@ -23,83 +23,23 @@ wss_url = "wss://api.pythtest.pyth.network"
 key_store.program_key = "RelevantOracleProgramAddress"
 
 # Public key of the root mapping account
 key_store.mapping_key = "RelevantOracleMappingAddress"
 
-# Optional public key of the accumulator program (if provided)
-key_store.accumulator_key = "RelevantOracleAccumulatorAddress"
+### Optional fields of primary/secondary network config ###
 
-### Optional fields ###
-
-# [metrics_server]
-#
-# Where to serve the quick-access dashboard and metrics. Metrics live under "/metrics"
-# NOTE: non-loopback addresses must be used carefully, making sure the
-# connection is not exposed for unauthorized access.
-# bind_address = "127.0.0.1:8888"
-
-# [remote_keypair_loader}
-# Where to serve the remote keypair loading endpoint, under "/primary/load_keypair" and "/secondary/load_keypair"
-# NOTE: non-loopback addresses must be used carefully, making sure the
-# connection is not exposed for unauthorized access.
-# bind_address = "127.0.0.1:9001"
-
-# How much whole SOL must a keypair hold to be considered valid for use on a given network. Disabled with 0
-# primary_min_keypair_balance_sol = 1
-# secondary_min_keypair_balance_sol = 1
-
-
-# Channel capacities. These refer to async messaging channels
-# internally used by the agent's subroutines
-
-# Capacity of the channel used to broadcast shutdown events to all
-# components
-# channel_capacities.shutdown = 10000
-
-# Capacity of the channel used to send updates from the primary Oracle
-# to the Global Store
-# channel_capacities.primary_oracle_updates = 10000
-
-# Capacity of the channel used to send updates from the secondary
-# Oracle to the Global Store
-# channel_capacities.secondary_oracle_updates = 10000
-
-# Capacity of the channel the Pythd API Adapter uses to send lookup
-# requests to the Global Store
-# channel_capacities.global_store_lookup = 10000
-
-# Capacity of the channel the Pythd API Adapter uses to communicate
-# with the Local Store
-# channel_capacities.local_store_lookup = 10000
-
-# Capacity of the channel on which the Local Store receives messages
-# channel_capacities.local_store = 10000
-
-# Capacity of the channel on which the Pythd API Adapter receives
-# messages
-# channel_capacities.pythd_adapter = 10000
-
-# Capacity of the slog logging channel. Adjust this value if you see
-# complaints about channel capacity from slog
-# channel_capacities.logger_buffer = 10000
-
-
-# Relative path to publisher identity keypair
-# w.r.t. `key_store.root_path`. When the specified file is not found
-# on startup, the relevant primary/secondary network will expect a
-# remote-loaded keypair. See remote_keypair_loader options for
-# details.
-# key_store.publish_keypair_path = "publish_key_pair.json" # I exist, remote loading disabled
-# key_store.publish_keypair_path = "none" # I do not exist, remote loading activated for the network
-
-# Relative path to accumulator message buffer program ID. Setting this
+# Pubkey of accumulator message buffer program ID. Setting this
```

Comment:
The older ordering of these options made additional `exporter.*` and `oracle.*` options fall under the `remote_keypair_loader` sections. This means that if someone were to uncomment all config options and sections, they would not end up with a correct config.

With this change, every config section is clearly marked and additional `{primary|secondary}_network` options exist in the correct spot, still under `[primary_network]`